drizzlepac/hlautils/sourcelist_generation.py

```diff
@@ -648,13 +140,11 @@ def create_sourcelists(obs_info_dict, param_dict):
     Returns
     -------
     """
-    log.info("----------------------------------------------------------------------------------------------------------------------")
+    log.info("-" * 118)
     for key1 in list(obs_info_dict.keys()):
         for key2 in list(obs_info_dict[key1].keys()):
```

Comment:
Consider using 'product_info_type' or 'product_info_label' to be more descriptive than 'key2'.

Reply:
issue resolved by commit [6e2cc21](https://github.com/spacetelescope/drizzlepac/pull/393/commits/6e2cc21453ab911dbf3800c32748c6b49cf9ebb1)